src/Components/Components/src/Forms/DataAnnotationsValidator.cs

```diff
@@ -19,7 +19,7 @@ protected override void OnInitialized()
             {
                 throw new InvalidOperationException($"{nameof(DataAnnotationsValidator)} requires a cascading " +
                     $"parameter of type {nameof(EditContext)}. For example, you can use {nameof(DataAnnotationsValidator)} " +
-                    $"inside an {nameof(EditForm)}.");
+                    $"inside an EditForm.");
```

Comment:
I guess this is a bit awkward with the new layering, but since it's just part of an exception message I'm not really worried. We can always change it in the future.

Reply:
This seemed OK to me. The idea of a the error message is to help people figure it out. Right now this is still the most likely problem.